### ISSUE TITLE
Display entire expected properties order in rule-properties-order message

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#539]
Example as scss-lint
```bash
PropertySortOrder: Properties should be ordered float, margin, border, border-radius, background, padding, width, text-align, text-decoration, line-height, color, font-size
```
All property and their order

**Comment by jeddy3** [comment]
Not sure how viable this is as `rule-properties-order`, unlike scsslint's `PropertySortOrder`, allows for _flexible groups_, so suggesting a specific order of properties, within the rule message, for the whole rule-set might be odd. So, then we probably need to question the usefulness of such a feature.

Free free to investigate the viability and usefulness of it though @evilebottnawi :)

**Comment by evilebottnawi** [comment]
@jeddy3 When I'm working in my ide much easier to see the whole list of the wrong sort than moving each property in the right place and then look for errors, and then move, and then look for errors) spend a lot of my time

**Comment by jeddy3** [comment]
That does seem useful, especially if you've specified your own order.

@davidtheclark What do you think? Open it out to a PR? I suspect there'll be a fair bit of work involved in making this change.

**Comment by davidtheclark** [comment]
Sure, wouldn't mind a PR.

**Comment by jeddy3** [comment]
@evilebottnawi Feel free to add this feature via a PR :)

FYI, the message should probably be something like `Expected property order to be "x", "y" and then "z"`, to keep it inline with [the conventions](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#messages).

**Comment by jeddy3** [comment]
@davidtheclark Shall we close this? In the age of [`postcss-sorting`](https://github.com/hudochenkov/postcss-sorting) I think there is little motivation to make this change, as I think most people will just be wanting a flag that says their ordering is wrong... so they know to run `postcss-sorting`.

**Comment by davidtheclark** [comment]
Yes, I'm closing.
